static/app/components/core/alert/index.chonk.tsx

```diff
@@ -95,6 +106,17 @@ function makeChonkAlertTheme(
   throw new TypeError(`Invalid alert type, got ${type}`);
 }
 
+function generateAlertBackground(
+  props: ChonkAlertProps,
+  tokens: ReturnType<typeof getChonkAlertTokens>
+) {
+  const width = 44;
```

Comment:
Can users set icon sizes larger than this, what would happen if they do?

Reply:
Nope, the leading icon is hardcoded and not configurable. I checked that this also works with browser zoom and it does.